sources/Algo/src/Algo.cpp

```diff
@@ -0,0 +1,40 @@
+//
+// Copyright (c) 2020, RTE (http://www.rte-france.com)
+// See AUTHORS.txt
+// All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, you can obtain one at http://mozilla.org/MPL/2.0/.
+// SPDX-License-Identifier: MPL-2.0
+//
+
+/**
+ * @file  Algo.cpp
+ *
+ * @brief Dynaflow launcher algorithms implementation file
+ *
+ */
+
+#include "Algo.h"
+
+#include <tuple>
+
+namespace dfl {
+namespace algo {
+
+SlackNodeAlgorithm::SlackNodeAlgorithm(std::shared_ptr<inputs::Node>& slackNode) : slackNode_{slackNode} {}
```

Comment:
Do we plan to add all algorithms into the same file?

Reply:
Yes. For me we can put all algorithms in the same file, since they seems small

Reply:
OK, let's keep this for now but if the file becomes too large we will need to split it